prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html

```diff
@@ -80,6 +80,9 @@
             <mat-error *ngIf="licenseNumber.hasError('alphanumeric')">
               Must contain only alphanumeric characters
             </mat-error>
+            <mat-error *ngIf="licenseNumber.hasError('numeric') || licenseNumber.hasError('length')">
+              Must contain only 5 numeric characters
+            </mat-error>
```

Comment:
How about using a 2-in-1 pattern validator:
```HTML
<mat-error *ngIf="licenseNumber.hasError('pattern')">
```
and 
```typescript
licenseNumberValidators.push(Validators.pattern('^[0-9]{5}$'));
```

Reply:
What is the benefit of say doing it this way?

Reply:
I agree with the self descriptive part. However the validation message is triggered by `numeric` or `length`, the user does not really know which rule they violated. It generated the same outcome as the pattern validator unless there are two separate validation messages `Must contain numeric` and `Must contain 5 chars` which gives the user more insights.

Reply:
Yeah I agree having two messages would probably be more descriptive to the user, but we were asked to follow what we did for the CPSID or whatever the other one is, so the functionality was copied over from there and didn't change anything.